Loger/Starter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Loger
+{
+    static class Starter
+    {
+        private static int _logCount = 100;
+        public static void Run()
+        {
+            for (int i = 0; i <= _logCount; i++)
+            {
+                int rand = new Random().Next(1, 4);
```

Comment:
You can create new Random() before the loop (var randomizer = new Random();) and then assign it to a variable and use randomizer. Next(1, 4); inside for.